message/utils/checkIfEmpty.js

```diff
@@ -1,4 +1,6 @@
-const checkIfEmpty = (str) => {
+const checkIfEmpty = (input) => {
+
+    const str = input + ''
```

Comment:
What are you trying to accomplish here?
What will happen if the input is nulll?

Reply:
I'm trying to check if the string or the number received is empty, as long as it should display a different error when it is empty or when it is false (an empty string i considered in js as a falsy, so it has to be checked by a different method)